### PR TITLE
[Chore] Improve Bamboo Bitbucket Jira Setup documentation

### PR DESCRIPTION
**Pull request by derLalla**
### Description

This improves the documentation of setting up Jira in the Bamboo/Bitbucket local setup. Jira Core has become outdated and license keys for it are not available any more.

### CODE REVIEW

**Review by nicolasruscher — APPROVED** [review]
I have not tested it, but the text itself looks good to me, apart from the one GitHub annotation, but I do not think its a big problem :D

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 1]
```diff
@@ -75,19 +75,31 @@ under ``localhost:7990``.
...
+   Also, you can select the evaluation/internal/test/dev setups if you are asked.
+   Follow the additional steps for Jira and Bitbucket.
+
+   - Jira:
+    - On startup select ``I'll set it up myself``
+    - Select Build In Database Connection
+    - Create a sample project
+
+   - Bitbucket: Do not connect Bitbucket with Jira yet
+
```
Comment (SimonEntholzer) [inline docs/dev/setup/bamboo-bitbucket-jira.rst:94]: Maybe add a step that makes sure that `xdg-utils` are installed. We found an issue that occured when the `xdg-open` command was not found.
```suggestion
3. Make sure that `xdg-utils` is installed before running the script. `xdg-utils <https://www.howtoinstall.me/ubuntu/18-04/xdg-utils/>`
```

**Review by SimonEntholzer — CHANGES_REQUESTED** [review]
Looks good, found one possible improvement. :)

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 2]
```diff
@@ -75,19 +75,31 @@ under ``localhost:7990``.
...
+
+3. Execute the shell script ``atlassian-setup.sh`` in the
+   ``src/main/docker`` directory (e.g. with
+   ``src/main/docker/./atlassian-setup.sh``). This script creates
+   groups, users ([STRIKEOUT:and adds them to the created groups] NOT
+   YET) and disabled application links between the 3 applications
```
Comment (derLalla) [inline docs/dev/setup/bamboo-bitbucket-jira.rst:100]:
```suggestion
   YET) and disabled application links between the 3 applications. Make sure that ``xdg-utils`` is installed before running the script. `xdg-utils <https://www.howtoinstall.me/ubuntu/18-04/xdg-utils/>`__
```

File: docs/dev/setup/bamboo-bitbucket-jira.rst [hunk 3]
```diff
@@ -75,19 +75,31 @@ under ``localhost:7990``.
...
+    - Create a sample project
+
+   - Bitbucket: Do not connect Bitbucket with Jira yet
+
+
+3. Execute the shell script ``atlassian-setup.sh`` in the
+   ``src/main/docker`` directory (e.g. with
+   ``src/main/docker/./atlassian-setup.sh``). This script creates
+   groups, users ([STRIKEOUT:and adds them to the created groups] NOT
+   YET) and disabled application links between the 3 applications. Make sure that ``xdg-utils`` is installed before running the script. `xdg-utils <https://www.howtoinstall.me/ubuntu/18-04/xdg-utils/>`__
```
Comment (derLalla) [inline docs/dev/setup/bamboo-bitbucket-jira.rst:100]:
```suggestion
   YET) and disabled application links between the 3 applications. Make sure that `xdg-utils <https://www.howtoinstall.me/ubuntu/18-04/xdg-utils/>`__ is installed before running the script.
```

**Review by nicolasruscher — APPROVED** [review]
approving the new changes 👍

**Review by SimonEntholzer — APPROVED** [review]
Looks good now :+1:

**Pull request merged**